src/common/pages/apiDocumentation/api_documentation.scss

```diff
@@ -0,0 +1,132 @@
+@import '../../../style/colors';
+.api-page-container{
+    .search-field{
+        width: 500px;
+        display: flex;
+        margin: auto;
+        margin-bottom: 25px;
+        ::before{
+            display: none ;
+        }
+        input{
+            padding: 10px !important;
+        }
+        .MuiInputBase-input {
+            border: 1px solid #e0e0e0 !important;
+         
+        }
+        @media (max-width:768px){
+            width: 100%;
+        }
+    }
+    .accordion{
+        .accordion-body{
+            word-break: break-all;
+            .pre-raw-format{
+                display: block;
+                white-space: pre-wrap;
+                word-break: break-word;
+                background: transparent;
+                border: none;
+                margin: 0;
+                padding: 0 0 0.5em;
+            }
+            table{
+                max-width: 95%;
+                & th, td{
+                    border: 1px solid #706f70;
+                border-collapse: collapse;
+                    padding: 5px;
+                    word-break: break-word;
+                }
+            }
+        }
+        .accordion-button{
+            display: flex;
+            justify-content: space-between;
+            &:focus{
+                box-shadow: none !important;
+            }
+            // &::after{
+            //     display: none;
+            // }
+        }
+        &.accordion-day{
+            .accordion-header{
+                .accordion-button{
+                    background-color: #f5f5f5;
+                  
+                }
+            }
+            table{
+                th{
+                    background-color: #f5f5f5;
+                }
+            }
+            .text-p{
+                color: #333333;
+            }
+        }   
+        &.accordion_night{
+            background-color: transparent;
+            .accordion-item{
+                background-color: transparent;
+                color: #fff;
+                .accordion-button{
+                    background: $dark-grey-blue;
+                    color: #fff;
+                    &::after{
+                       
+                       background-image: url('../../img/chevron-down-white.svg') !important;
```

Comment:
better utilize and move this file to `/common/img/svg.tsx` to unify all images into single place and delete .svg file itself.